knexfile.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
       host: 'localhost',
       user: 'euniceem1007',
       password: 'password',
-      database: 'react-kanban'
+      database: 'euniceem1007'
```

Comment:
We should make sure you get setup on having a database for the application and not use you username as a database.

Ask us to help you also move these settings to your `.env` file and then we can pull from `process.env` so that we're not committing sensitive information to our repository.